gym/vector/async_vector_env.py

```diff
@@ -362,20 +362,25 @@ def set_attr(self, name, values):
         name : string
             Name of the property to be set in each individual environment.
 
-        values : list or object
-            Values of the property to bet set to. If `values` is a list, then
+        values : list of object
```

Comment:
Should this be `or` like before (suggested by the description and by the code), or `of` as changed to now? If the latter, I think the description should be updated to reflect that change. Should the source code raise an exception on non-list types, too?

Reply:
No this was a mistake indeed, I thought it was a typo but it is list *or* object.